### DISCUSSION TITLE
Link click is not fire on `::before|::after` pseudo elements

### DISCUSSION THREAD

**Discussion by bozdevs** [swup/swup#973] [Help]
Link click is not fire on `::before|::after` pseudo elements and Swup visit doesn't start.
Is there a need or a logical explanation for this? Otherwise, we can consider it as a BUG.

**Comment by hirasso** [comment]
Could you please provide a minimal reproduction of the described behavior.

**Comment by daun** [comment]
As far as I know, pseudo elements are part of the element they belong to -- at least in the context of event handling. You can't set or unset click handlers on them. So I'd guess this is a styling issue, but as @hirasso noted, it needs a reproduction :)
